addons/block_code/ui/picker/categories/category_factory.gd

```diff
@@ -404,6 +404,46 @@ static func get_general_blocks() -> Array[Block]:
 	b.category = "Math"
 	block_list.append(b)
 
+	b = BLOCKS["parameter_block"].instantiate()
+	b.block_name = "randf"
+	b.variant_type = TYPE_FLOAT
+	b.block_format = "Random number between 0.0 and 1.0"
+	b.statement = "randf()"
+	b.defaults = {}
+	b.category = "Math"
+	b.tooltip_text = "Generate a random floating point number between 0.0 and 1.0 inclusively ([0.0, 1.0])"
+	block_list.append(b)
+
+	b = BLOCKS["parameter_block"].instantiate()
+	b.block_name = "randf_range"
+	b.variant_type = TYPE_FLOAT
+	b.block_format = "Random number between {from: FLOAT} and {to: FLOAT}"
+	b.statement = "(randf_range({from}, {to}))"
+	b.defaults = {"from": -1.0, "to": 1.0}
```

Comment:
Maybe the 2 "random floating point" blocks can be one, if the defaults are `from: 0.0, to: 1.0`. Then the code (statement) can be:
```
randf() if {from} == 0.0 and {to} == 1.1 else randf_range({from}, {to})
```